app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
```

Comment:
- [ ] I think your app will be more engaging if we can have few other details (name, bio and photo) as listed in the schema.rb as part of the sign-up parameters. 👍 

## Your project
<img width="824" alt="Untitled" src="https://github.com/Shakir-Hussain12/blog_app/assets/103386183/1daff302-8e9d-4f59-a3ab-0b34bd1d8a0f">

<img width="904" alt="Untitled" src="https://github.com/Shakir-Hussain12/blog_app/assets/103386183/54630c68-e8ef-4738-beb5-2270a1ff8e59">

**The bio, image and username was autogenerated by the app, but I think the user should be able to do this at the point of registration (sign-up)** 👍